image-customize

```diff
@@ -89,7 +89,7 @@ class InstallAction(ActionBase):
         if os.path.isfile(package):
             pname = os.path.basename(package)
             dest = "/var/tmp/" + pname
-            machine_instance.upload([package], dest)
+            machine_instance.upload([os.path.abspath(package)], dest)
```

Comment:
OK, but this needs checking our existing projects that they don't rely on the old behaviour. I did a quick grep, and it should be okay.

Reply:
I'm reasonably certain that this would absolutely always work.  We only ever enter this block if `os.path.isfile()` found the named file, relative to the current working directory, and in that case `os.path.abspath()` is going to definitely give the same result.

The only time this might break down is if we have a file called `a`, and also a file called `test/a` and we specify to upload `a`, which is then found in the current directory, but we actually wanted `test/a`.  I can't imagine that anyone is making use of [this sort of thing](https://xkcd.com/1172/)...

Reply:
You are right, thanks!